osu.Game.Rulesets.Osu/OsuDifficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -95,7 +95,7 @@ private void computeSliderCursorPosition(Slider slider)
             var computeVertex = new Action<double>(t =>
             {
                 // ReSharper disable once PossibleInvalidOperationException (bugged in current r# version)
-                var diff = slider.PositionAt(t) - slider.LazyEndPosition.Value;
+                var diff = slider.PositionAt(t) + slider.StackOffset - slider.LazyEndPosition.Value;

```

Comment:
How about we expose `StackedPositionAt(double time)` from `Slider`? We'll probably want to use this elsewhere.